__tests__/components/buttons/ClearButton.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { fireEvent, render, screen } from '@testing-library/react-native';
+import React from 'react';
+import ClearButton from '../../../components/buttons/ClearButton';
+
+describe('<ClearButton/>', () => {
+  // Test that <ClearButton/> component is rendered
+  it('renders clearbutton component', () => {
+    render(<ClearButton onPress={() => console.log('Powered off')} label="Power off" iconName="poweroff" />);
+    expect(screen.getByTestId('clearButton')).toBeTruthy();
+  });
+
+  // Test that the label press is shown.
+  it('renders label poweref off', () => {
```

Comment:
This looks like a typo. Shouldn't it be `power` instead of `poweref`?

Reply:
Yes, it should, let me correct it